website/docs/r/notification_channel.html.markdown

```diff
@@ -109,27 +117,28 @@ resource "newrelic_notification_channel" "foo" {
 ##### [Jira Classic](https://docs.newrelic.com/docs/apis/nerdgraph/examples/nerdgraph-api-notifications-channels/#jira)
 ```hcl
 resource "newrelic_notification_channel" "foo" {
+  account_id = 1
   name = "jira-example"
   type = "JIRA_CLASSIC"
   destination_id = "00b6bd1d-ac06-4d3d-bd72-49551e70f7a8"
```

Comment:
Do you think we should also include an additional example showing how to create a destination resource and then reference that destination the channel resource using?

i.e.
```hcl
destination_id = newrelic_notification_destination.foo.id
```

Reply:
Also, the more real world examples we can provide, the better. We don't need something for every single scenario, but as long as the examples are actual working examples, we should be good.

Reply:
sure! I'll add more examples for real scenarios